eli5/keras/explain_prediction.py

```diff
@@ -104,38 +152,71 @@ def explain_prediction_keras(model, # type: Model
     -------
       expl : :class:`eli5.base.Explanation`
         An :class:`eli5.base.Explanation` object for the relevant implementation.
+
+        The following attributes are supported by all concrete implementations:
+
+        * ``targets`` a list of :class:`eli5.base.TargetExplanation` objects \
+            for each target. Currently only 1 target is supported.
+        * ``layer`` the hidden activation layer that we chose to do computations with, \
+            as a string (name of the layer).
```

Comment:
I think layer can also be an int and a Layer instance, and it's good that we support it, so maybe it would be better to remove or fix the type information here ("as a string").

Reply:
I can add support for int.

But I am not sure about Layer instance @lopuhin . I think I will use this `layer` argument with PyTorch as well, but it has a different type for a "layer" (`nn.Module`). The resulting type would be something like `Union[str, int, Any]`?

Reply:
Maybe we can say that type of "layer" argument is backend-specific?

Reply:
Sounds good to me. So do you want the type annotation to just be `Any`? In this case I will always pass the layer instance where possible instead of the name / id.

Reply:
Yes, I think `Any` would be a fine annotation here, and passing they layer sounds good 👍